examples/image_classification_kornia.ipynb

```diff
@@ -0,0 +1,3856 @@
+{
```

Comment:
<p>In this notebook, we'll fine-tune from the <a href="https://huggingface.co/microsoft/swin-tiny-patch4-window7-224" target="_blank">https://huggingface.co/microsoft/swin-tiny-patch4-window7-224</a> checkpoint, but note that there are many, many more available on the <a href="https://huggingface.co/models?other=vision" target="_blank">hub</a>. We will also use the <a href="https://huggingface.co/docs/datasets/installation" target="_blank">datasets</a> library to load an image dataset .</p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>